Exceptions/EntityNotFoundException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Infrastructure\Exceptions;
+
+
+use Symfony\Component\Routing\Exception\ResourceNotFoundException;
+
+class EntityNotFoundException extends ResourceNotFoundException
+{
+
+}
```

Comment:
Why you don't use this exception "HttpResourceNotFoundException" ?

Reply:
because it is not a http mapper